cms/static/js/factories/export.js

```diff
@@ -24,17 +24,26 @@ define(['gettext', 'js/views/feedback_prompt'], function(gettext, PromptView) {
                 }
             });
         } else {
-            var msg = '<p>' + gettext('There has been a failure to export your course to XML. Unfortunately, we do not have specific enough information to assist you in identifying the failed component. It is recommended that you inspect your courseware to identify any components in error and try again.') + '</p><p>' + gettext('The raw error message is:') + '</p>' + errMsg;
+            var msg = '<p>';
+            var action;
+            if (library) {
+                msg += gettext('There has been a failure to export your library to XML. Unfortunately, we do not have specific enough information to assist you in identifying the failed component. It is recommended that you inspect your library to identify any components in error and try again.');
+                action = gettext('Yes, take me to the main library page')
```

Comment:
I agree that having "Yes" on the button text is a bit strange, since there is no corresponding question in the error message. 
I'd also like to suggest a rewrite for the error message (for both library and course export). 
Suggested text: "Your library could not be exported to XML. There is not enough information to identify the failed component. Inspect your library to identify any problematic components and try again."
Suggested revision for button text: "Take me to the main library page".
@marcotuts @cahrens @Kelketek do you agree?